HISTORY.md

```diff
@@ -60,6 +60,9 @@ makes a couple of minor usability improvements, such as better error checking.
 - Raise error if `result_archive` and `archive` have different fields
   ({pr}`461`)
 - Warn user if resampling for bounds takes too long in ESs ({pr}`462`)
+- Implement BOP-Elites by adding BayesianOptimizationEmitter and
+  BayesianOptimizationScheduler ({pr}`496`)
+- Add BOP-Elites demo
```

Comment:
This should go under `Documentation` in the `(Forthcoming)` release as well. It should also have the PR number on it:
```
({pr}`496`)
```